blazor-toc.html

```diff
@@ -4076,17 +4091,6 @@
 			</li>
 		</ul>
 	</li>
-	<li>Sankey
```

Comment:
Why removed it @VigneshwaranGovindharajan  ?

Reply:
Added removed lines.